build_examples_test.go

```diff
@@ -0,0 +1,38 @@
+package git
```

Comment:
We don't need this file, as `make test` already does the trick:

```
$ make test
go test  -v _examples/common_test.go _examples/common.go --examples
=== RUN   TestExamples
=== RUN   TestExamples/blame
git clone https://github.com/git-fixtures/basic.git /tmp/go-git-blame-3662081995
git blame CHANGELOG
b8e471f5 (Daniel Ripolles 2015-03-31 13:44:52 +0200 1) Initial changelog
=== RUN   TestExamples/branch
git clone https://github.com/git-fixtures/basic.git /tmp/3841571656
git branch my-branch
git branch -D my-branch
=== RUN   TestExamples/checkout
git clone https://github.com/git-fixtures/basic.git /tmp/3011666005
git show-ref --head HEAD
6ecf0ef2c2dffb796033e5a02219af86ec6584e5
git checkout 35e85108805c84807bc66a02d91535e1e24b38b9
git show-ref --head HEAD
35e85108805c84807bc66a02d91535e1e24b38b9
=== RUN   TestExamples/checkout-branch
git clone https://github.com/git-fixtures/basic.git /tmp/1284450122
git show-ref --head HEAD
6ecf0ef2c2dffb796033e5a02219af86ec6584e5
git checkout branch
local checkout of branch 'branch' failed, will attempt to fetch remote branch of same name.
like `git checkout <branch>` defaulting to `git checkout -b <branch> --track <remote>/<branch>`
checked out branch: branch
git show-ref --head HEAD
...
```

Reply:
Yes, I have removed the file